src/modeler/AutoRest.Swagger/Validation/Core/RulesExtensions.cs

```diff
@@ -64,6 +64,6 @@ public static IEnumerable<Rule> GetValidationRules(this Type type)
         /// <summary>
         /// The collection of rules that apply to all properties regardless of other rules.
         /// </summary>
-        public static IEnumerable<Rule> UniversalRules = new[] { new NoControlCharacters() };
+        public static IEnumerable<Rule> UniversalRules = new List<Rule>();
```

Comment:
A. excellent
B. well my main point is that "universal" is a weird concept and I bet it is always just "almost universal" and really just flawed
C. sure, no problem, just thought since this project isn't really supposed to grow anymore, we can cut dead code - but we can wait until the TS project is solid

but that also reminds me: didn't you say you also tried triggering this rule? did you succeed? if no, universal rules may not be implemented correctly anyways, so there is an opportunity to fix a bug by removing the buggy code 😆